.github/workflows/docs-spelling.yml

```
@@ -36,21 +36,29 @@ name: Check Spelling
     tags-ignore:
-    - "**"
+      - '**'
+    paths:
+      - 'packages/docs/**'
+      - '.github/workflows/docs-spelling.yml'
+      - '.github/actions/docs-spelling/**'
   pull_request_target:
     branches:
-    - "**"
+      - '**'
     tags-ignore:
```

actionlint (1.7.8)

"tags-ignore" filter is not available for pull_request_target event. it is only for push event

(events)